integrated_channels/integrated_channel/models.py

```diff
@@ -926,12 +926,6 @@ def __str__(self):
             f", status_code: {self.status_code}"
         )
 
-    def __repr__(self):
-        """
-        Return uniquely identifying string representation.
-        """
-        return self.__str__()
-
```

Comment:
is it intentionally removed?

Reply:
no it was not intentionally removed?
updated.